### PR TITLE
Doc 11144: Example import filter

### PR DESCRIPTION
**Pull request by franrob-projects**
- adds new examples and descriptions
- adds to the configuration/import filter page
- adds to sync/import processes page

Jira: [DOC-11144](https://issues.couchbase.com/browse/DOC-11144)

### CODE REVIEW

File: modules/ROOT/pages/import-processing.adoc [hunk 1]
```diff
@@ -24,5 +24,53 @@ Refer to the xref:configuration-schema-import-filter.adoc[Import Filter Configur
...
+
+| name 
+| name 
+| Name of sync gateway database
+
+| scopes 
+| tenant1 
+| Name of scope is tenant1
+
+| Collections:
```
Comment (osfameron) [inline modules/ROOT/pages/import-processing.adoc]: Is this the right name?

Feels like you'd want to describe the `import` function here?
e.g. perhaps something like `scopes.{scope}.collections.{collection}.import` ?
(Also is "Collections" with a capital `C` correct?)

File: modules/ROOT/pages/import-processing.adoc [hunk 2]
```diff
@@ -24,5 +24,53 @@ Refer to the xref:configuration-schema-import-filter.adoc[Import Filter Configur
...
+| name 
+| name 
+| Name of sync gateway database
+
+| scopes 
+| tenant1 
+| Name of scope is tenant1
+
+| Collections: 
+| Collection1: import: function(doc) {// customize}
```
Comment (osfameron) [inline modules/ROOT/pages/import-processing.adoc]: This doesn't look right. I think the value here is the import function, so just `function(doc) {// customize}`
(which is code and should be in backticks)

File: modules/ROOT/pages/configuration-schema-import-filter.adoc [hunk 3]
```diff
@@ -23,16 +23,64 @@ It's worth noting that Sync Gateway imports all documents by default, so it's ge
...
 == Configuration
 
+[source,JSON]
+----
+{
+   "bucket": "bucket",
+   "name": "name",
+   "scopes": {
+       "tenant1": {
+           "Collections": {
```
Comment (osfameron) [inline modules/ROOT/pages/configuration-schema-import-filter.adoc]: again, is `Collections` with capital C correct here?

File: modules/ROOT/pages/configuration-schema-import-filter.adoc [hunk 4]
```diff
@@ -23,16 +23,64 @@ It's worth noting that Sync Gateway imports all documents by default, so it's ge
...
+
+| name 
+| name 
+| Name of sync gateway database
+
+| scopes 
+| tenant1 
+| Name of scope is tenant1
+
+| Collections:
```
Comment (osfameron) [inline modules/ROOT/pages/configuration-schema-import-filter.adoc]: See comments below

File: modules/ROOT/pages/configuration-schema-import-filter.adoc [hunk 5]
```diff
@@ -23,16 +23,64 @@ It's worth noting that Sync Gateway imports all documents by default, so it's ge
...
+| scopes 
+| tenant1 
+| Name of scope is tenant1
+
+| Collections: 
+| Collection1: import: function(doc) {// customize} 
+| The default sync function assigns all documents to the "Collection1" channel. There is also a custom import filter that is defined specifically for this collection.
+|===
+
+
```
Comment (osfameron) [inline modules/ROOT/pages/configuration-schema-import-filter.adoc:63]: These lines 72-80 have been copied from openapi-generator output, and are messy/not needed, DELETE

File: modules/ROOT/pages/configuration-schema-import-filter.adoc [hunk 6]
```diff
@@ -23,16 +23,64 @@ It's worth noting that Sync Gateway imports all documents by default, so it's ge
...
+// tag::delete_keyspace-_config-import_filter-responses[]
+
+// markup not found, no include::{specDir}paths/delete_keyspace-_config-import_filter/operation-responses-begin.adoc[opts=optional]
+
+
+
 
 [#ex-config]
-.Using an Import Filter
```
Comment (osfameron) [inline modules/ROOT/pages/configuration-schema-import-filter.adoc:35]: Ah, is this a misleading title?

**Review by osfameron — CHANGES_REQUESTED** [review]
Content looks mostly good - I think the table needs tweaking to make the import filter value/description clearer, and you have some random copy/pasted artefacts that need deletion 😸

File: modules/ROOT/pages/import-processing.adoc (hunk 1 above)
Comment (franrob-projects) [reply modules/ROOT/pages/import-processing.adoc]: Ahh -- tenent1 = scope1

File: modules/ROOT/pages/import-processing.adoc (hunk 2 above)
Comment (franrob-projects) [reply modules/ROOT/pages/import-processing.adoc]: agreed

File: modules/ROOT/pages/configuration-schema-import-filter.adoc (hunk 5 above)
Comment (franrob-projects) [reply modules/ROOT/pages/configuration-schema-import-filter.adoc:63]: cool

File: modules/ROOT/pages/configuration-schema-import-filter.adoc (hunk 6 above)
Comment (franrob-projects) [reply modules/ROOT/pages/configuration-schema-import-filter.adoc:35]: For me, it was just a touch random as it displayed:

Example 1. blah

and we have more examples on the same subject

File: modules/ROOT/pages/configuration-schema-import-filter.adoc [hunk 7]
```diff
@@ -61,25 +61,14 @@ If you are using legacy configuration then, you need to include it in your confi
 | Name of sync gateway database
 
 | scopes 
-| tenant1 
-| Name of scope is tenant1
+| Scope1 
+| Name of scope is Scope1
 
 | Collections:
```
Comment (osfameron) [inline modules/ROOT/pages/configuration-schema-import-filter.adoc]: This name is still wrong isn't it?

Comment (osfameron) [reply modules/ROOT/pages/configuration-schema-import-filter.adoc]: e.g. `Collections:`
We're not explaining what a "Collections" looks like, but an import filter?
(or am I misunderstanding)

Comment (franrob-projects) [reply modules/ROOT/pages/configuration-schema-import-filter.adoc]: Yeah I get what you are saying

**Review by osfameron — APPROVED** [review]
:shipit: 🚢

**Pull request merged**